<launch>
    <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop_twist_keyboard" output="screen">
        <remap from="cmd_vel" to="/robot_0/cmd_vel"/>
    </node>
</launch>